dbms/src/Interpreters/Join.cpp

```diff
@@ -2337,13 +2349,30 @@ void Join::checkTypes(const Block & block) const
     checkTypesOfKeys(block, sample_block_with_keys);
 }
 
+void Join::RowsNotInsertToMap::insertRow(Join::RowRefList * elem, Block * stored_block, size_t index, bool need_materialize)
+{
+    insertRowToList(&head, elem, stored_block, index);
```

Comment:
will the RowRefList be used in nullaware join, looks like we don't need to keep the RowRefList for nullaware join?

Reply:
Good point!